### PR TITLE
fixed NULL input for log level. Added -fPIC to build CPP files

### PR DESCRIPTION
**Pull request by smtakeda**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -36,7 +36,7 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 if (UNIX)
     # Linux and OSX
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -Werror")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -std=gnu++0x")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -std=gnu++0x -fPIC")
```
Comment (smtakeda) [inline CMakeLists.txt]: @howryu @notkriswagner do you have any specific reason why you didn't add -fPIC? Without this, the PHP PDO build will fail.

Comment (howryu) [reply CMakeLists.txt]: Go ahead and add it. No reasons for me.

**Comment by howryu** [comment]
LGTM. Not related to this pr, but I have a general question. 

Should we add ability to capture logs from curl, aws sdk in the same log file as snowflakel logs? If yes, we need to provide some interface. Also maybe addd namespace for each log entry.

**Comment by smtakeda** [comment]
yes, we need curl log as well. Can you file a issue including more specific needs?

**Comment by smtakeda** [comment]
@howryu https://github.com/snowflakedb/libsnowflakeclient/issues/7 is the place to add comments about logging.

**Pull request merged**